pkg/version/version.go

```diff
@@ -149,6 +128,17 @@ func GetVersionBundle(version string) (VersionBundle, error) {
 		MulticlusterControlplane: "v0.4.0",
 	}
 
+	versionBundleList["0.14.0"] = VersionBundle{
+		Registration:             "0.14.0",
+		Placement:                "0.14.0",
+		Work:                     "0.14.0",
+		Operator:                 "0.14.0",
+		AddonManager:             "0.14.0",
+		AppAddon:                 "0.14.0",
+		PolicyAddon:              "0.14.0",
```

Comment:
Why are there no `v` prefixes here? Is that handled elsewhere?

Reply:
updated